client/my-sites/domains/domain-management/style.scss

```diff
@@ -665,13 +665,29 @@ ul.email-forwarding__list {
 	margin-top: 5px;
 }
 
+.add-email-addresses-card__name-fieldsets {
+	margin-top: 20px;
+
+	@include breakpoint( '>660px' ) {
+		display: flex;
+		flex-direction: row;
```

Comment:
I think `row` is the default value for `flex-direction`, so we shouldn't have to specify it.

Reply:
done